template/.claude/commands/tdd-review.md

```
@@ -0,0 +1,110 @@
+---
+description: Review test suite quality against FIRST principles and TDD anti-patterns
+argument-hint: [optional test file or directory path]
+---
+
+## General Guidelines
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)